src/stores/user.ts

```diff
@@ -82,9 +90,9 @@ export const useUserStore = defineStore("userStore", {
       }
     },
     async getUser() {
-      this.callLogout();
       try {
         const response = await getUser();
+        console.log(response);
```

Comment:
I've seen several console.logs, are we sure we want them here? 🤔

Reply:
No excuse here, need to improve me clean up game :)